stable/insights-agent/templates/pluto/rbac.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.pluto.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-pluto
+  labels:
+    app: insights-agent
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-pluto
+  labels:
+    app: insights-agent
+rules:
+  - apiGroups:
+      - 'apps'
+    resources:
+      - 'deployments'
+    verbs:
+      - 'get'
+      - 'list'
+      - 'watch'
+  - apiGroups:
+      - ''
+    resources:
+      - 'namespaces'
+      - 'pods'
+      - 'secrets'
+      - 'configmaps'
+    verbs:
+      - 'get'
+      - 'list'
+      - 'watch'
+  - apiGroups:
+      - 'autoscaling.k8s.io'
+    resources:
+      - 'verticalpodautoscalers'
```

Comment:
Looks like this RBAC was copied from Goldilocks - I think a better RBAC profile is probably the one release-watcher has: https://github.com/FairwindsOps/charts/blob/master/stable/insights-agent/templates/release-watcher/rbac.yaml

That gives
* cluster-view
* read access to secrets

Reply:
Got it. Thanks for catching that. I also see that release-watcher is using a newer apiVersion too so I updated those as well. I adopted the use of the `view` clusterrole for the pluto serviceaccount, and created a separate clusterrole for being able to view secrets. I kept namespaces (because Helm 3 is namespace aware so I thought it might be needed) and configmaps as the Pluto team said that was necessary.